dotcom-rendering/src/web/components/Snap.tsx

```diff
@@ -19,12 +19,7 @@ export const Snap = ({ snapData }: Props) => {
 	return (
 		<>
 			<div
-				css={[
-					snapStyles,
-					css`
-						${snapData.embedCss}
-					`,
-				]}
+				css={[snapStyles]}
```

Comment:
We can now remove `${snapData.embedCss}` from the `Snap`. We only needed this because the DOM for the Cotton Capital thrashers was `<Snap snapData={trail.snapData} />` instead of: 

```
<SnapCssSandbox snapData={trail.snapData}>
	<Section
		<Snap snapData={trail.snapData} />
	</Section>
</SnapCssSandbox>
```